shell/platform/darwin/macos/framework/Source/FlutterEmbedderExternalTextureUnittests.mm

```diff
@@ -18,6 +19,47 @@
 #include "third_party/skia/include/core/SkSamplingOptions.h"
 #include "third_party/skia/include/core/SkSurface.h"
 
+@interface TestExternalTexture : NSObject <FlutterTexture>
```

Comment:
Just double checking, all TUs are ARC enabled right? Reviewing with that assumption.

Reply:
Yup, they are